lib/model/folder_sendrecv.go

```diff
@@ -387,6 +387,9 @@ func (f *sendReceiveFolder) processNeeded(snap *db.Snapshot, dbUpdateChan chan<-
 			}
 
 		case runtime.GOOS == "windows" && file.IsSymlink():
+			if err := f.handleSymlinkCheckExisting(file, snap, scanChan); err != nil {
+				f.newPullError(file.Name, fmt.Errorf("handling unsupported symlink: %w", err))
+			}
```

Comment:
return or continue or something?

Reply:
maybe also a `// No reason to retry`, `changed--` as in one place above?

Reply:
Do we really know here that a retry won't help? In the case above it's an invalid file, that will never work. But here it might be e.g. a case-conflicting file that is about to be deleted in the pull later on.